www/js/menu.js

```diff
@@ -304,7 +305,7 @@ module.exports = {
     const $preferencesOpen = document.querySelectorAll('.preferences-open');
     $preferencesOpen.forEach(($menuToggle) => {
       $menuToggle.appendChild(menuButton.cloneNode(true));
-      $menuToggle.addEventListener('click', () => { module.exports.toggleMenu('#menu-page'); });
+      $menuToggle.addEventListener('click', module.exports.showDisplayTab);
```

Comment:
> Keep the settings icon in presenter view. When you click it, it will open the display tab on the bottom right

do we need the settings icon in presenter view? its pretty far away mouse wise from what it opens in presenter view. it has some benefits in non-presenter view.

Reply:
Me and @inderpreetsingh had a discussion on this. We are unsure if people are using this button to access the settings. So rather than getting rid of it completely and confuse people, we are leaving it in place for now. However yes, that is the plan, to eventually remove this button as it is no longer needed in Presenter View

Reply:
I am not happy with the UI we have at the moment either. I especially don't like how menu behaves in non-presenter view. However, that's all base don opinions, I would rather use real-world data to decide how to move further.

Reply:
I would like to play around with this once we have a new alpha is released with this. Let's discuss after that.

Reply:
Okay! I will push a quick separate PR and a video for what I think is the ideal solution. 🙏